### DISCUSSION TITLE
Lesson 5: Running the Aderyn Command

### DISCUSSION THREAD

**Discussion by kwamentw** [Cyfrin/security-and-auditing-full-course-s23#16] [Ideas]
i use Kali linux and when i installed the current version of aderyn his command did not work for me 

Note: make sure you are in the root directory of your project
aderyn now takes the root path as a default arg so no need to pass the --root 

```bash
aderyn .
```

we use the dot because that is an easy way of passing your current directory as an argument

Upvote if this helped

**Comment by cromewar** [comment]
@kwamentw 
Most of devs sadly don't use linux (at least not mid range devs) I'm glad someone is using Kali over here, thanks for the tip!

**Reply by kwamentw** [reply]
I understand you cos I go through alot to find solutions online.
